src/main/kotlin/com/github/psxpaul/task/JavaExecFork.kt

```diff
@@ -78,4 +90,11 @@ open class JavaExecFork @Inject constructor(forkOptionsFactory: JavaForkOptionsF
         val maxCommandLineLength = if (DefaultNativePlatform.getCurrentOperatingSystem().isWindows()) 32767 else 131072
         return args.joinToString(" ").length > maxCommandLineLength
     }
+
+    private fun getEffectiveExecutable(): String {
```

Comment:
Analog to https://github.com/gradle/gradle/blob/master/subprojects/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java#L777